src/ignitus-Shared/Components/ProfileCard/Styles/index.scss

```diff
@@ -0,0 +1,168 @@
+$base-color: #000066;
+$white-bg: #fff;
+
+.list-item {
+  padding: 2em 1em;
+  display: flex;
+  flex-wrap: wrap;
+  box-shadow: 0 2px 5px 5px rgba(0,0,0,0.1);
```

Comment:
Put box-shadow as this `box-shadow: 0 1px 8px 1px rgba(0, 0, 0, 0.1); `
Make this card CSS in a shared component.

Reply:
I have put this card in shared component . By "Make this card CSS in a shared component " what exactly you mean?